card/db/migrate_core_cards/20150708224756_add_list_cards.rb

```diff
@@ -4,19 +4,26 @@ class AddListCards < Card::CoreMigration
   def up
     create_card! name: 'List', codename: :list,
                  type_code: :cardtype
+
     create_card! name: 'Listed by', codename: :listed_by,
                  type_code: :cardtype
-    create_card! name: '*cached count', codename: :cached_count,
-                 subcards: {
-                    '+*right+*update'=>'[[Administrator]]',
-                    '+*right+*create'=>'[[Administrator]]',
-                    '+*right+*delete'=>'[[Administrator]]'
-                  }
-    create_card! name: '*cached content', codename: :cached_content,
-                 subcards: {
-                   '+*right+*update'=>'[[Administrator]]',
-                   '+*right+*create'=>'[[Administrator]]',
-                   '+*right+*delete'=>'[[Administrator]]'
-                 }
+
+    create_or_update name: '*cached count',
+                     codename: :cached_count,
+                     rename_if_conflict: false,
+                     subcards: {
+                        '+*right+*update'=>'[[Administrator]]',
+                        '+*right+*create'=>'[[Administrator]]',
+                        '+*right+*delete'=>'[[Administrator]]'
+                      }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.